components/spinner.py

```diff
@@ -2,30 +2,129 @@
 import rev
 import rev.color
 
+class Colour: #class for storing rgb colours
+    def __init__(self, red: float, green: float, blue: float):
+        self.red = red
+        self.green = green
+        self.blue = blue
+
+    def dist(self, c) -> tuple:
+        return (
+            abs(self.red - c.red) + abs(self.green - c.green) + abs(self.blue - c.blue)
+        )
 
 class Spinner:
-    spinner_motor: wpilib.Spark
-    spinner_solenoid: wpilib.DoubleSolenoid
-    colour_sensor: rev.color.ColorSensorV3
+	spinner_motor: wpilib.Spark
```

Comment:
Inconsistent indentation. Please ensure your indentation is consistent within the same file.

Reply:
(Preferably, run black.)

Reply:
I have now run black